src/service/persistence.js

```diff
@@ -211,12 +213,37 @@ async function addCommentToHelpRequest(externalSystemId, fields) {
     }
 }
 
+async function addCommentToHelpRequestResolve(externalSystemId, { category, how} ) {
```

Comment:
would you be able to add a test for this function and the addLabel function to https://github.com/hmcts/slack-help-bot/blob/main/src/service/persistence.functional.test.js

we run those when Jira is getting upgraded to make sure everything still works. and it's quicker to develop with those tests than by running through the slack-help-bot all the time

Reply:
Test functions added for addCommentToHelpRequestResolve and addLabel functions and tested using jest.